public/app/models/concerns/tree_nodes.rb

```diff
@@ -28,7 +28,15 @@ def breadcrumb
   def breadcrumb_identifier(record, type)
     case type
     when 'resource'
-      resolved_resource['id_0'] if resolved_resource
+      if resolved_resource
+        id_0 = resolved_resource['id_0']
```

Comment:
@avatar382 an alternative could be `resolved_resource.slice('id_0', 'id_1', 'id_2', 'id_3').values.compact.join('-')` ... but if you want to leave it as is I think we're good here, works as expected.